docs/how-to-guides/rook-ceph-storage.md

```diff
@@ -0,0 +1,293 @@
+# Storage with Rook Ceph on Packet cloud
+
+## Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Deploy storage worker pool](#deploy-storage-worker-pool)
+  - [Config](#config)
+  - [Deploy the worker pool](#deploy-the-worker-pool)
+- [Deploy Rook](#deploy-rook)
+  - [Config](#config-1)
+  - [Deploy the component](#deploy-the-component)
+- [Deploy Rook Ceph](#deploy-rook-ceph)
+  - [Config](#config-2)
+  - [Deploy the component](#deploy-the-component-1)
+- [Access the Ceph dashboard](#access-ceph-dashboard)
+- [Enable and access toolbox](#enable-and-access-toolbox)
+- [Enable monitoring](#enable-monitoring)
+- [Make default storage class](#make-default-storage-class)
+- [Additional resources](#additional-resources)
+
+## Introduction
+
+This guide provides the steps for deploying a storage stack using the `rook` and `rook-ceph` Lokomotive component and explains how to access Ceph dashboard, Ceph toolbox and how to enable monitoring.
+
+## Prerequisites
+
+* A Lokomotive cluster deployed on a supported provider and accessible via `kubectl`.
```

Comment:
In the title we talk about Packet, but here is just a supported provider? Does the guide works on AWS too then?

Reply:
Good catch. Made it Packet specific.